include/swift/AST/PrintOptions.h

```diff
@@ -425,6 +425,7 @@ struct PrintOptions {
     result.AbstractAccessors = false;
     result.PrintForSIL = true;
     result.PrintInSILBody = true;
+    result.PreferTypeRepr = false;
```

Comment:
The SIL tests and that one ide test were depending on the fact that VarDecls were always printed using the Type (they were checking the output as printed when using the type).
I preferred to make the dependency on type printing explicit instead of updating all the tests, which for SIL printing I think using type printing makes more sense.